src/core/config/mac.h

```diff
@@ -318,12 +318,12 @@
  * @def OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
  *
  * This setting configures the CSL receiver feature in Thread 1.2.
- * This is compulsory for 1.2 MTD, optional for 1.2 FTD.
+ * Set it to 0 by default because it is also used in 'platform' which cannot access the core config. To enable this
+ * option, it has to be set in build options when building.
```

Comment:
```suggestion
```
I think users already know they need to specify build options when building.

Reply:
Make sense. Has removed it.